requirements.txt

```diff
@@ -1 +1 @@
-Theano==0.7
+git+https://github.com/Theano/Theano.git@4dacb9e7d04d842d8ffe2c948dbc90722ae990d4#egg=Theano
```

Comment:
I think that's a discussion that deserves its own issue. Previously there was an implicit agreement that we would standardize on version 0.7 as soon as it was released. If we're going to change our stance on this we should discuss it thoroughly first.